pkgs/by-name/li/librespeed-go/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+let
+  version = "1.1.5";
+  src = fetchFromGitHub {
+    owner = "librespeed";
+    repo = "speedtest-go";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ywGrodl/mj/WB25F0TKVvaV0PV4lgc+KEj0x/ix9HT8=";
+  };
+in
+buildGoModule {
+  pname = "librespeed-go";
+  inherit version src;
+
+  vendorHash = "sha256-ev5TEv8u+tx7xIvNaK8b5iq2XXF6I37Fnrr8mb+N2WM=";
+
+  ldflags = [
+    "-w"
+    "-s"
```

Comment:
This strips all symbols from the binary. It might make sense to disable this if `dontStrip` is set, although I am not sure how this functionality is exposed in `buildGoModule`.